src/docs/content/observability/observability-scenario/pre-requisites/setup.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 1
+tags:
+  - mushop setup
+---
+
+## Pre-requisites
+
+### Install Ingress Controller
```

Comment:
Create an ingress manually to a standard deployment (Which already includes an Ingress Nginx) will cause conflicts and may break the solution. This also is not the same package, causing more disturbance on the deployment.

Reply:
we need to deploy ingress controller before applying mushop.yaml. which then applies the rules from ingress to edge.